lib/grizzly/options.ex

```diff
@@ -4,6 +4,20 @@ defmodule Grizzly.Options do
   alias Grizzly.Supervisor
   alias Grizzly.ZIPGateway.Config
 
+  @type rf_region() ::
+          :eu
+          | :us
+          | :australia_new_zealand
+          | :hong_kong
+          | :malaysia
+          | :india
+          | :isreal
+          | :russia
+          | :china
+          | :us_lr
+          | :japan
+          | :korea
```

Comment:
I'm not 100% sure this matters, but zipgateway likes to use abbreviations for all of them. The only thing that's a little weird to me is that the eu and us are abbreviated and everything else is spelled out.

```
enum
{
  EU = 0x00,  /* Europe */
  US = 0x01,  /* US */
  ANZ = 0x02, /* Australia & New Zealand */
  HK = 0x03,  /* Hong Kong */
  MY = 0x04,  /* Malaysia */
  ID = 0x05,  /* India */
  IL = 0x06,  /* Israel */
  RU = 0x07,  /* Russia */
  CN = 0x08,  /* China */
  US_LR = 0x09,  /* US Long Range*/
  JP = 0x20,  /* Japan */
  KR = 0x21   /* Korea */
};
```

Something else that's interesting is that in `railtest` (I randomly was looking at the railtest docs this afternoon), Malaysia has been deleted and there's a note that you're not supposed to use code 4 any more.